modules/nw-multinetwork-sriov.adoc

```diff
@@ -40,247 +60,379 @@ and device ID 0x1015
+<7> The number of the virtual functions is to be created for each SR-IOV
+physical network device.
+<8> The NIC selector selects the NIC to be configured. You don't have to specify
+all the fields. However it is recommended to select the NICs as specifically as
+possible, in order to minimize the possibility of overlapping cross policies.
+And if you specify both `pfName` and `rootDevices` at the same time, which is
+recommended, please make sure they point to the identical devices.
+<9> The vendor hex code of SR-IoV device. Allowed value "8086", "15b3".
+<10> The device hex code of SR-IoV device. Allowed value "158b", "1015", "1017".
+<11> The names of SR-IoV physical function.
+<12> The PCI addresses of SR-IoV physical function. 
+<13> The driver type of the virtual functions. Allowed value "netdevice",
+"vfio-pci". Defaults to "netdevice".
+<14> The RDMA mode. Defaults to false. In this release, only RoCE mode are
+supported, and only for Mellanox NICs. 
 
-. Run the following command to create the `sriov-conf` CR:
-+
+== Configuring SR-IOV networks
+
+The SR-IOV Network Operator also introduces a CRD `SriovNetwork` for creating
+the Custom Resource of `NetworkAttachmentDefinition` of SR-IOV CNI plugin. When
+you create a CR of `SriovNetwork`, the operator will created a CR of
+`NetworkAttachmentDefinition` accordingly.
+
+You should never have to modify this CRD. Instructions for creating or 
+modifying a CR are provided in this documentation as appropriate.
+
+[NOTE]
+=====
+You shall not modify or delete a Custom Resource of `SriovNetwork`, when it has
+been used by any running Pods.
+=====
+
+The following is an example of a typical Custom Resource for
+`SriovNetwork`.
+
+[source,yaml]
 ----
-$ oc create -f sriov-conf.yaml
+apiVersion: sriovnetwork.openshift.io/v1
+kind: SriovNetwork
+metadata:
+  name: sriov-conf <1>
+  namespace: openshift-sriov-network-operator <2>
+spec:
+  networkNamespace: default <3>
+  ipam: | <4>
+    {
+      "type": "dhcp"
+    }
+  vlan: 0 <5>
+  resourceName: sriov <6>
 ----
+<1> The name of the CR. The generated `NetworkAttachmentDefinition` CR will use
+the same name.
+<2> The namespace of the CR, this must be in the same namespace of operator.
+<3> The namespace where the `NetworkAttachmentDefinition` CR will be created.
+<4> The IPAM configuration for SR-IOV CNI plugin.
+<5> The VLAN ID for SR-IOV CNI plugin. Range from 0 to 4095, default to 0.
+<6> The SRIOV Network device plugin endpoint resource name. It shall matches
+the `resourceName` defined in the `SriovNetworkNodePolicy` CR.
+
+== Configuring additional interfaces using SR-IOV
 
 . Create a YAML file for a Pod which references the name of the
 `NetworkAttachmentDefinition` and requests one `openshift.io/sriov` resource:
 +
+[NOTE]
+=====
+The SR-IoV admission controller will inject the `resource` field automatically
+if a NetworkAttachmentDefinition CR of SRIOV CNI plugin is referred in the Pod
+annotation.
+=====
++
```

Comment:
Perhaps adding pod examples for both dpdk and rdma mode.

Reply:
Added.